### ISSUE TITLE
Remove deprecated t.python.otp

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5493]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#5493|
|Type|task|
|Created|2012-02-19 21:11:40Z|
|Branch|https://github.com/twisted/twisted/tree/del-otp-5493|

[twisted/python/otp.py](twisted/python/otp.py) was deprecated in 9.0 (r25457) and can be deleted.

<details><summary>Searchable metadata</summary>

```
trac-id__5493 5493
type__task task
reporter__thijs thijs
priority__low low
milestone__ 
branch__branches_del_otp_5493 branches/del-otp-5493
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1329685900000000 1329685900000000
changetime__1331141243000000 1331141243000000
version__None None
owner__thijs thijs
cc__thijs cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra commented|
|-|-|

(In [33581]) Branching to 'del-otp-5493'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [33582]) Remove deprecated otp module, refs [#5493](https://github.com/twisted/twisted/issues/5493)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

Up for review. (note to self: [#2053](https://github.com/twisted/twisted/issues/2053) tries to completely get rid of `test_import`)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_5"></a>|@jesstess set owner to @thijstriemstra|
|-|-|

Thanks for working on this, thijs.

Your changes look good and [buildbot results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/del-otp-5493) look good, so please merge!

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [33658]) Merge del-otp-5493: Removed twisted.python.otp, deprecated since 9.0.

Author: thijs
Reviewer: jesstess
Fixes: [#5493](https://github.com/twisted/twisted/issues/5493)
